example/templates/free-shipping.html

```diff
@@ -2,6 +2,7 @@
   <label>
     <input type="checkbox" {{#isRefined}}checked{{/isRefined}} />
     {{label}}
+    {{_refined}}!{{/refined}}
```

Comment:
This was just to illustrate that the options worked, and I didn't want to do it with the `delimiters` options, because it would have meant changing all delimiters in *example/templates/**. It's no different from `isRefined`, the only difference is that it's using the section tags of `Hogan` (you can see there is no `#` in the opening tag).
We can remove it.